lib/controllers/cats.js

```diff
@@ -1,20 +1,18 @@
 const { Router } = require('express');
+const Cat = require('../models/Cat');
 
-const { cats } = require('../../data/cats')
 
 
-const router = Router();
 
-router
-.get('/:id', (req, res) => {
-    const id = req.params.id;
-    const matchingCat = cats.find((cat) => cat.id === id);
+module.exports = Router()
+  .get('/:id', async (req, res) => {
+    const matchingCat = await Cat.getById(req.params.id);
     res.json(matchingCat);
-})
+  })
+
+  .get('/', async (req, res) => {
+    const cats = await Cat.getAll();
```

Comment:
You have a full list of cats here, with more properties than we want - you can .map through your cats array, and end up with an array that only has the properties of `id` and `name`